### ISSUE TITLE
Accessibility: Inline frame without a text alternative

### ISSUE THREAD

**Issue by kevinkidwell** [cul-it/exhibits-library-cornell-edu#526]
**Describe the bug**
When a video is embedded using an iFrame HTML element, you must add a title attribute that states the title of the video, if it is not present. This should be fixed by Content Owners. Example below of correctly formatted title attribute:

> <iframe width="800" height="574" src="https://macaulaylibrary.org/asset/33501031/embed/800" frameborder="0" allowfullscreen style="width:800px;" **title="This is your title of the video"**></iframe>

Exhibit: FASHION & FEATHERS
Page: https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58323110240&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r13/failed/aG90OE5kdFE=;ZDc2ZDdiZmI=
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58323110825&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-

Exhibit: Women Empowered: Fashions from the Frontline
https://my2.siteimprove.com/Inspector/1081272/A11Y/Page?pageId=58330560599&impmd=3CC33A985F387DC37EA87D3D5A2306C4&decision=dismissed+falsePositive+none#/sia-r13/failed/dzNKekExZ0w=;OTQ4NjRhNTE=
